fcrepo-service-activemq/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -4,25 +4,41 @@
        xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
        xsi:schemaLocation="
        http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0 http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
-       http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
-       http://camel.apache.org/schema/blueprint http://camel.apache.org/schema/blueprint/camel-blueprint.xsd">
+       http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
 
    <!-- OSGI blueprint property placeholder -->
    <cm:property-placeholder id="properties" persistent-id="org.fcrepo.camel.service.activemq" update-strategy="reload">
      <cm:default-properties>
        <cm:property name="jms.brokerUrl" value="tcp://localhost:61616"/>
        <cm:property name="jms.username" value=""/>
        <cm:property name="jms.password" value=""/>
+       <cm:property name="jms.connections" value="10"/>
```

Comment:
If you intentionally did not update the `fcrepo-service-activemq#README.md` with these low-level configuration additions, that is fine. But I wanted to raise it here in case that was a simple oversight.